public/scripts/project.js

```diff
@@ -1,149 +1,155 @@
 'use strict';
 
-function Project(name, description, url, backgroundimage) {
-  this.name = name;
-  this.description = description;
-  this.url = url;
-  this.backgroundimage = backgroundimage;
-}
-
-Project.all = [];
-
-Project.prototype.toHtml = function() {
-  let template = Handlebars.compile($('#project-template').text());
-  return template(this);
-};
-
-Project.load = function(projectData){
-  projectData.forEach(function(proj) {
-    Project.all.push(new Project(proj.name, proj.description, proj.url, proj.backgroundimage));
-  })
-}
-
-Project.fetch = function() {
-  if(localStorage.projectData){
-    Project.load(JSON.parse(localStorage.projectData));
-    projectView.initIndexPage();
-  } else {
-    $.ajax({
-
-      url: './data/project_file.json',
-      dataType: 'json',
-      function (err) {
-        console.error(err)
-      },
-    }).done(function(data) {
-      localStorage.setItem('projectData', JSON.stringify(data));
-      Project.load(JSON.parse(localStorage.projectData));
-      projectView.initIndexPage();
-    })
+((module) => {
+
+  function Project(name, description, url, backgroundimage) {
+    this.name = name;
```

Comment:
But just a suggestion. This works out perfect, just something to think about when creating objects that have multiple properties.